examples/bert/yamls/test/main.yaml

```diff
@@ -3,11 +3,11 @@
 #   `composer main.py yamls/test/main.yaml` to run using the HuggingFace BERT
 #   `composer main.py yamls/test/main.yaml model.name=mosaic_bert` to run using the Mosaic BERT
 
-data_remote: &data_remote ./my-copy-c4 # Follow the instructions in the README to set up ./my-copy-c4
-data_local: &data_local ./my-copy-c4
-tokenizer_name: &tokenizer_name prajjwal1/bert-tiny
-max_seq_len: &max_seq_len 128
-mlm_probability: &mlm_probability 0.15
+data_remote: null # Follow the instructions in the README to set up ./my-copy-c4
```

Comment:
did you intentionally change this to null?

Reply:
yeah if you're not streaming, ie if ds is local, remote should be set to null (see [here](https://github.com/mosaicml/examples/blob/main/examples/llm/yamls/mosaic_gpt/125m.yaml#L1))
we should probably that comment to the line that defines `data_local`

Reply:
Yes, this was a bug from previous streaming usage where local and remote were set to the same value. Now, we set remote to None